clusterloader/docs/design.md

```diff
@@ -48,7 +48,7 @@ A single test scenario will be defined by a `Config`. Its schema will be as foll
 struct Config {
 	// TBD: Some config allowing to access cluster, e.g. path to kubeconfig file.
 
-	// Number of namespace in the test.
+	// Number of (default) namespaces in the test.
```

Comment:
I don't understand. Isn't 'default' namespace only one?

Reply:
I guess it's overloaded term.
I didn't mean the k8s 'default' namespace.

I meant a "default set of namespace" that you will get out of the box already created, deleted at the end and managed for you.

Do you suggest different name?

Reply:
I read below. The use of 'default' is a bit confusing here. Maybe say something like 'No. of namespaces you want the test to create'?

Reply:
Changed to sth different - PTAL.